src/main/kotlin/heist/make/sdk/kt/event/EventTarget.kt

```diff
@@ -1,10 +1,13 @@
 package heist.make.sdk.kt.event
 
+import kotlinx.serialization.Serializer
+
 
 /**
  * The EventTarget class impplements methods that add and remove event listeners to type-specific events.
  */
 
+@kotlinx.serialization.Serializable
```

Comment:
You dont need the fully qualified path here. 
```
@Serializable // That should suffice
```

Reply:
I just wrote @Serializable and the IDE changed it to the full path